pkg/analyzer/malware.go

```
@@ -78,35 +78,32 @@ func (a *malwareAnalyzer) Analyze(manifest *models.PackageManifest,
 			return nil
 		}
 
-		var filterMsg string
+		// When Malware with Verification Record or Suspicious with Paranoid Mode Enabled
 		if pkg.IsMalware() {
```

@KunalSin9h Should we simplify this by returning if not malware. 

The rest of the code for reporting policy violation event and fail fast become simpler. 

We don't need to check for malware again during fail fast

---

aah!, simplified!